docs/data/material/components/autocomplete/autocomplete.md

```diff
@@ -227,6 +227,11 @@ The `renderInput` prop allows you to customize the rendered input.
 The first argument of this render prop contains props that you need to forward.
 Pay specific attention to the `ref` and `inputProps` keys.
 
+:::warning
+If you are using custom input component to render in Autocomplete, pay much attention for `ref` prop.
+You might need forwardRef for your custom input and you should pass `ref` to `inputRef` when using mui's `TextField`.
```

Comment:
Here's how I would phrase this (if I'm understanding it correctly):

```suggestion
If you're using a custom input component inside the Autocomplete, you might need to include `forwardRef`. 
When incorporating the Text Field component, you should pass `ref` to `inputRef`.
```

Reply:
Yes, now it looks better